README.md

```diff
@@ -47,7 +49,7 @@ Source code is available [here](https://github.com/pleerock/class-transformer-de
 - [Samples](#samples)
 - [Release notes](#release-notes)
 
-## What is class-transformer
+## What is class-transformer[⬆](#table-of-contents)<!-- Link generated with jump2header -->
```

Comment:
Could you regenerate it with `--silent` to hide those comments?

Reply:
> could you regenerate it with --silent to hide those comments?

Yes

> Please also add a space before the icon, if possible.

If space is added then slug will become `#release-notes-` instead of `#release-notes` and navigation will be broken.
But there are also options to insert the link below the header or at the end of header section.